src/models/link/mod.rs

```diff
@@ -33,8 +33,6 @@ use crate::models::{VirtualTargetPath, TargetDescription};
 //     )
 // }
 
-pub const FILENAME_FORMAT: &str = "{step_name}.{keyid:.8}.link";
-
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Link {
     // Why is the type named as typ?
```

Comment:
We had the _type field here. The serde rename statement now applies to the name field? If so, it may be messing up the structure of the final metadata?